Description: test started failing in January 2020 apparently string
 handling changed
Author: Paul Gevers <elbrus@debian.org>

Index: uglifyjs/test/compress/reduce_vars.js
===================================================================
--- uglifyjs.orig/test/compress/reduce_vars.js
+++ uglifyjs/test/compress/reduce_vars.js
@@ -2063,7 +2063,7 @@ issue_1814_2: {
             }(0);
         }();
     }
-    expect_stdout: "0 '321'"
+    expect_stdout: "0 321"
 }
 
 try_abort: {
